### PR TITLE
Kits/Manager: Fix - Show Globals dropdown to all of the user.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: core/kits/assets/js/manager.js [hunk 1]
```diff
@@ -242,11 +229,13 @@ export default class extends elementorModules.editor.utils.Module {
 				this.renderGlobalsDefaultCSS();
 			} );
 
-			elementor.on( 'panel:init', () => {
-				this.addPanelPages();
+			elementor.hooks.addFilter( 'controls/base/behaviors', this.addGlobalsBehavior );
 
-				this.addPanelMenuItem();
-			} );
+			if ( ! elementor.config.user.can_edit_kit && ! elementor.config.user.is_administrator ) {
```
Comment (matipojo) [inline core/kits/assets/js/manager.js:234]:
```suggestion
			if ( ! elementor.config.user.can_edit_kit ) {
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/assets/js/manager.js (hunk 1 above)
Comment (iNewLegend) [reply core/kits/assets/js/manager.js]: Handled, force-pushed.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

File: core/kits/assets/js/manager.js [hunk 2]
```diff
@@ -242,11 +229,13 @@ export default class extends elementorModules.editor.utils.Module {
...
-				this.addPanelPages();
+			elementor.hooks.addFilter( 'controls/base/behaviors', this.addGlobalsBehavior );
 
-				this.addPanelMenuItem();
-			} );
+			if ( ! elementor.config.user.can_edit_kit ) {
+				return;
+			}
+
+			$e.components.register( new Component( { manager: this } ) );
```
Comment (iNewLegend) [inline core/kits/assets/js/manager.js:238]: test

File: core/kits/assets/js/component.js [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
+			args.manager.addPanelPages();
+
+			args.manager.addPanelMenuItem();
+
+			elementor.hooks.addFilter( 'panel/header/behaviors', args.manager.addHeaderBehavior );
+		} );
+
+		elementor.addControlView( 'global-style-repeater', Repeater );
+	}
+
```
Comment (iNewLegend) [inline core/kits/assets/js/component.js:21]: test

**Pull request merged**